packages/server/index.js

```diff
@@ -18,12 +18,13 @@ function createIMAServer({
   emitter,
   performance,
   devUtils,
+  applicationFolder,
```

Comment:
Maybe we could set the default value in here? `applicationFolder: defaultApplicationFolder` So we don't have to resolve the correct value twice.